src/cljs/ataru/virkailija/application/application_subs.cljs

```diff
@@ -679,7 +675,7 @@
 (re-frame.core/reg-sub
   :application/eligibility-automatically-checked?
   (fn [db _]
-    (let [hakukohde (get-in db [:application :selected-review-hakukohde])]
+    (let [hakukohde (get-in db [:application :selected-review-hakukohde-oids])]
       (->> (get-in db [:application :events])
```

Comment:
Tää ei futaa tällassenaan. Tässäkin varmaan pitäis kattoa, että päteehän tää ehto kaikille valituille hakukohteille?